biggraphite/drivers/cassandra.py

```diff
@@ -964,25 +944,60 @@ def __init__(
         self.__shard = self.__shard = bg_accessor.pack_shard(replica, writer)
         self.__shard = c_marshal.int16_unpack(c_marshal.uint16_pack(self.__shard))
 
-        if self.use_lucene:
-            self.metadata_query_generator = lucene.CassandraStratioLucene(
-                self.keyspace_metadata,
-                self.max_queries_per_pattern,
-                self.max_metrics_per_pattern,
-            )
-        else:
-            self.metadata_query_generator = sasi.CassandraSASI(
-                self.keyspace_metadata,
-                self.max_queries_per_pattern,
-                self.max_metrics_per_pattern,
+        # Metadata settings
+        self._enable_metadata = enable_metadata
+        self.metadata_query_generator = None
+        self.keyspace_metadata = None
+        self.contact_points_metadata = None
+        self.port_metadata = None
+        self._meta_write_consistency = None
+        self._meta_read_consistency = None
+        self._meta_serial_consistency = None
+        self._meta_background_consistency = None
+        self.__cluster_metadata = None  # setup by connect()
+        self.__session_metadata = None  # setup by connect()
+        self.__select_metric_metadata_statement = None  # setup by connect()
+        self.__update_metric_metadata_statement = None  # setup by connect()
+        self.__touch_metrics_metadata_statement = None  # setup by connect()
+        self.__metadata_touch_ttl_sec = None
+        if enable_metadata:
```

Comment:
Even when disabled it could be worth setting most of the attributes (as long as nothing actually connects to cassandra)